<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerWidget</class>
 <widget class="QWidget" name="CustomerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>414</height>
   </rect>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>721</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="3">
     <widget class="QToolButton" name="select4">
      <property name="text">
       <string>select</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="symbol3Path">
      <property name="text">
       <string>Leer</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Symbol 5.</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLabel" name="symbol7Path">
      <property name="text">
       <string>Leer</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QLabel" name="symbol9Path">
      <property name="text">
       <string>Leer</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_5">
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Symbol 1.</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="symbol2Path">
      <property name="text">
       <string>Leer</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="symbol5Path">
      <property name="text">
       <string>Leer</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_6">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Symbol 2.</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QToolButton" name="select2">
      <property name="text">
       <string>select</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QToolButton" name="select1">
      <property name="text">
       <string>select</string>
      </property>
     </widget>
    </item>
    <item row="9" column="2">
     <widget class="QLabel" name="symbol10Path">
      <property name="text">
       <string>Leer</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QLabel" name="symbol8Path">
      <property name="text">
       <string>Leer</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QToolButton" name="select3">
      <property name="text">
       <string>select</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Symbol 11.</string>
      </property>
     </widget>
    </item>
    <item row="10" column="2">
     <widget class="QLabel" name="symbol11Path">
      <property name="text">
       <string>Leer</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Symbol 10</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Symbol 9.</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Symbol 12.</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Symbol 8.</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QToolButton" name="select5">
      <property name="text">
       <string>select</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QToolButton" name="select6">
      <property name="text">
       <string>select</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Symbol 7.</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Symbol 4.</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Symbol 6.</string>
      </property>
     </widget>
    </item>
    <item row="11" column="2">
     <widget class="QLabel" name="symbol12Path">
      <property name="text">
       <string>Leer</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="symbol6Path">
      <property name="text">
       <string>Leer</string>
      </property>
     </widget>
    </item>
    <item row="11" column="3">
     <widget class="QToolButton" name="select12">
      <property name="text">
       <string>select</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="symbol4Path">
      <property name="text">
       <string>Leer</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QToolButton" name="select7">
      <property name="text">
       <string>select</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QToolButton" name="select8">
      <property name="text">
       <string>select</string>
      </property>
     </widget>
    </item>
    <item row="8" column="3">
     <widget class="QToolButton" name="select9">
      <property name="text">
       <string>select</string>
      </property>
     </widget>
    </item>
    <item row="9" column="3">
     <widget class="QToolButton" name="select10">
      <property name="text">
       <string>select</string>
      </property>
     </widget>
    </item>
    <item row="10" column="3">
     <widget class="QToolButton" name="select11">
      <property name="text">
       <string>select</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Symbol 3.</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="symbol1Path">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Leer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
